config/default.yml

```diff
@@ -151,6 +151,18 @@ Sorbet/KeywordArgumentOrdering:
   Enabled: true
   VersionAdded: 0.2.0
 
+Sorbet/ObsoleteStrictMemoization:
+  Description: >-
+    This cop checks for the obsolete pattern for initializing instance variables that was required for older Sorbet
+    versions in `#typed: strict` files.
+
+    It's no longer required, as of Sorbet 0.5.10210
+    See https://sorbet.org/docs/type-assertions#put-type-assertions-behind-memoization
+  Enabled: pending
```

Comment:
We haven't been releasing major versions yet and from what I'm reading `pending` will stay disabled unless there is a major bump or `NewCops` setting is enabled. Given this should we enable it by default?